ui/app/mainui/activitycenter/views/ActivityNotificationCommunityKicked.qml

```diff
@@ -0,0 +1,52 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+import shared 1.0
+import shared.panels 1.0
+import shared.controls 1.0
+import utils 1.0
+
+import "../controls"
+
+ActivityNotificationBase {
+    id: root
+
+    bodyComponent: RowLayout {
+        readonly property var community: root.store.getCommunityDetailsAsJson(notification.communityId)
+
+        StatusSmartIdenticon {
+            id: identicon
+            name: community.name
+            asset.width: 40
+            asset.height: 40
+            asset.color: community.color
+            asset.letterSize: width / 2.4
+            asset.name: community.image
+            asset.isImage: true
+            Layout.leftMargin: Style.current.padding
+        }
+
+        StyledText {
+            text: qsTr("You have kicked from community")
+            font.weight: Font.Medium
+            font.pixelSize: 13
+            Layout.alignment: Qt.AlignVCenter
+        }
+
+        CommunityBadge {
+            communityName: community.name
+            communityImage: community.image
+            communityColor: community.color
+            onCommunityNameClicked: root.store.setActiveCommunity(notification.communityId)
+            Layout.alignment: Qt.AlignVCenter
+        }
+
+        Item {
```

Comment:
Why this empty space?

Reply:
To make items take minimum width, if there is no Layout.fillWidth item RowLayout stretches all the items